tests/specs/serve/class_instance/main.ts

```diff
@@ -0,0 +1,20 @@
+(async () => {
+  for (let i = 0; i < 1000; i++) {
+    try {
+      const resp = await fetch("http://localhost:12345/");
+      Deno.exit(0);
+    } catch {
+      await new Promise((r) => setTimeout(r, 10));
+    }
+  }
+
+  Deno.exit(2);
+})();
+
+class Foo implements Deno.ServeDefaultExport {
+  fetch(_request: Request) {
+    return new Response("Hello world!");
+  }
+}
```

Comment:
Maybe let's try updating this test to use something on the instance? For example, something like:

```
class Foo implements Deno.ServeDefaultExport {
  message = "Hello world!";
  fetch(_request: Request) {
    return new Response(this.message);
  }
}
```

I think right now it will not work because we need to bind the function to the instance.